test/runtime/samples/dynamic-element-string/_config.js

```diff
@@ -1,3 +1,7 @@
 export default {
-	html: '<div>Foo</div>'
+	html: '<div>Foo</div>',
+
+	test({ assert, target }) {
+		assert.htmlEqual(target.innerHTML,	'<div>Foo</div>');
+	}
```

Comment:
This is not directly related to this PR but I noticed that assertion is missing in this test case, so I added it.

Reply:
i dont think you need the extra `assert` here, since there's already an assert for the `config#html` right before the `config#test`, see the test script:

https://github.com/sveltejs/svelte/blob/master/test/runtime/index.ts#L201-L205

Reply:
Yeah, I realized already it asserted by `html`. so I reverted it.🙏